### PR TITLE
Fix some minor bugs

### PR DESCRIPTION
**Pull request by chrhck**
Fix some minor bugs that showed up running flake8

### CODE REVIEW

**Comment by chrhck** [comment]
merge once the checks pass

**Comment by chrhck** [comment]
Flake8 now throws a lot of style errors. I propose we merge this PR and fix the style issues in a new branch.

File: skyllh/core/pdf.py [hunk 1]
```diff
@@ -22,7 +22,8 @@
     func_has_n_args,
     issequenceof,
     range,
-    typename
+    typename,
+    classname
```
Comment (martwo) [inline skyllh/core/pdf.py]: I prefer sorted imports. So classname should be imported higher in that list

File: skyllh/core/math.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 
 from __future__ import division
 
+from copy import deepcopy
```
Comment (martwo) [inline skyllh/core/math.py]: again, sorted imports!

**Comment by chrhck** [comment]
closes #14 #13 #11 #12

**Pull request merged**